pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularUpdates.java

```diff
@@ -13,6 +13,7 @@ public class SingularUpdates implements CustomTask {
 
     @Override
     public void init(Path cfgPath, Config studyCfg, Config varsCfg) {
+        tasks.add(new SingularFileUploadNotifications());
```

Comment:
@ssettipalli Are new tasks supposed to be added here?

Reply:
It is ok but would be better to add to patch-log.conf or just run the task separately.